projects/swarmd/swarmd/Makefile

```diff
@@ -1,15 +1,4 @@
-.PHONY: tag push
-all: push
+ORG?=linuxkitprojects
+include ../../../pkg/package.mk
```

Comment:
I think you'll need `NETWORK=1` because the `docker build` requires networking for git cloning